include/sbg_driver/message_wrapper.h

```diff
@@ -422,8 +439,8 @@ class MessageWrapper
    * Create a ROS standard TwistStamped message from SBG messages.
    * 
    * \template  T                   	Euler or Quat message.
-   * \param[in] ref_sbg_ekf_nav_msg	SBG-ROS Ekf Nav message.
-   * \param[in] ref_sbg_imu_msg		SBG-ROS IMU message.
+   * \param[in] ref_sbg_ekf_nav_msg		SBG-ROS Ekf Nav message.
+   * \param[in] ref_sbg_imu_msg			SBG-ROS IMU message.
    * \return                        	ROS standard TwistStamped message.
    */
   template <typename T>
```

Comment:
Why it's a template ? and why is this implemented in the header??

Reply:
It is templated to accept either Euler or Quaternion data instead of doing two times the same function and duplicate code. 
Usually templated functions are implemented in the header file. There's a nice way to move the definition in the cpp file so I'll just do that.

Reply:
Still don't think it's a good idea...
You should rather have two methods with different arguments and then create the DCM either based on the quat or euler angles.
Then you can call the same private method that do the rest of the compution for instance.
But because you have to process / create differently the DCM for quat or euler template are not a good option.
Of course it's related to my other reviews such as removing from sbgMatrix3 dependencies to sbg messages.